integration-test/1491-brazil-shields.py

```diff
@@ -0,0 +1,827 @@
+                'type': 'route',
+            }, ways=[20906150]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 20906150,
+                'shield_text': '602', 'network': 'BR:RS:state',
+                'all_networks': ['BR:RS:state'],
+                'all_shield_texts': ['602'],
+            })
+
+    def test_br_VRS(self):
+        import dsl
+
+        z, x, y = (16, 23312, 38227)
+
+        self.generate_fixtures(
+            dsl.is_in('BR', z, x, y),
+            # https://www.openstreetmap.org/way/26757190
+            dsl.way(26757190, dsl.tile_diagonal(z, x, y), {
+                'source': 'openstreetmap.org', 'ref': 'VRS-851',
+                'highway': 'secondary', 'oneway': 'yes',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 26757190, 'shield_text': '851',
+                'network': 'BR:RS:secondary',
+            })
+
+    def test_br_RSC(self):
+        import dsl
+
+        z, x, y = (16, 23450, 38314)
+
+        self.generate_fixtures(
+            dsl.is_in('BR', z, x, y),
+            # https://www.openstreetmap.org/way/25979338
+            dsl.way(25979338, dsl.tile_diagonal(z, x, y), {
+                'name': 'Rota do Sol', 'surface': 'paved',
+                'source': 'openstreetmap.org', 'oneway': 'no',
+                'ref': 'RSC-453', 'highway': 'primary',
+            }),
+            dsl.relation(1, {
+                'network': 'BR', 'ref': 'BR-453', 'route': 'road',
+                'wikipedia': 'pt:BR-453', 'source': 'openstreetmap.org',
+                'wikidata': 'Q2877442', 'type': 'route',
+            }, ways=[25979338]),
+            dsl.relation(2, {
+                'old_ref': 'RST-453', 'name': 'Rota do Sol', 'route': 'road',
+                'source:official_name': 'LO 11432/2000', 'type': 'route',
+                'official_name': 'Rota do Sol Euclides Triches',
+                'source': 'openstreetmap.org', 'ref': 'RSC-453',
+                'network': 'BR:RS',
+            }, ways=[25979338]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 25979338,
+                'shield_text': '453', 'network': 'BR',
+                'all_shield_texts': ['453', '453'],
+                'all_networks': ['BR', 'BR:RS:connecting']
+            })
+
+    def test_br_SPA(self):
+        import dsl
+
+        z, x, y = (16, 24194, 37330)
+
+        self.generate_fixtures(
+            dsl.is_in('BR', z, x, y),
+            # https://www.openstreetmap.org/way/4964220
+            dsl.way(4964220, dsl.tile_diagonal(z, x, y), {
+                'source': 'openstreetmap.org', 'ref': 'SPA-344/055',
+                'highway': 'primary',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads',
+            {'id': 4964220, 'shield_text': '344', 'network': 'BR:SP:access'})
```

Comment:
I'd drop the `:access` part.

Reply:
And I think lower numbers should sort before so `055` for the shield text?

https://en.wikipedia.org/wiki/SP-55

Reply:
I've gone with lower numbers sort first, updated in c68d6eb.